lib/inventory.ml

```diff
@@ -60,11 +60,11 @@ let strip_quotes v =
   else v
 
 let parse_inventory_entry line =
-  match Astring.String.cuts ~empty:false ~sep:"=" line with
-  | [k; v] ->
+  match Astring.String.cut ~sep:"=" line with
+  | Some(k, v) ->
     (* trim whitespace *)
     Some (k, v |> strip_quotes |> String.trim)
```

Comment:
This is kind of funny. We get rid of the quotes and then trim the string. I'd have probably done something like `Astring.(String.trim k, String.trim ~drop:(fun c -> Char.Ascii.is_white c || c == '\'') v)`. Anyways, that's independent on this PR